buildconfig/stubs/pygame/math.pyi

```diff
@@ -18,8 +18,10 @@ if sys.version_info >= (3, 9):
 else:
     from typing import Collection
 
+# Math functions
 _T = TypeVar('_T')
 def clamp(value: _T, min: _T, max: _T, /) -> _T: ...
+def lerp(a: float, b: float, percent: float) -> float: ...
```

Comment:
Just like the function above, this takes positional-only args so it must have a trailing `/`